resources/newssite/news-nuxt/data/index.js

```diff
@@ -1,22 +1,22 @@
-import { content as contentEn } from "~/data/en/content";
-import { content as contentJp } from "~/data/jp/content";
-import { content as contentAr } from "~/data/ar/content";
+import { content as contentEn } from "@/data/en/content";
```

Comment:
Just curious, why have you changed this?

Reply:
looks like nuxt supports both ways and this is consistent with next.js. 
At least it still worked after I copied and pasted it by accident 🤣 

I can revert that change though if there are concerns

Reply:
I couldn't find any doc about that except from a babel plugin that I don't think we're using... Whatever works works also for me here